src/test/java/org/broadinstitute/hellbender/tools/spark/bwa/BwaSparkIntegrationTest.java

```diff
@@ -60,5 +61,4 @@ public void testSingleEnd() throws Exception {
 
         SamAssertionUtils.assertSamsEqual(new File(output, "part-r-00000.bam"), expectedSam);
     }
-
```

Comment:
Aren't there any additional tests for `BwaSpark` that pass with this fix and failed without it? If not, can you add one?